README.md

```diff
@@ -52,8 +57,7 @@ services:
     restart: always
     volumes:
       - /var/run/docker.sock:/tmp/docker.sock:ro # needs socket to read events
-      - ./certs/acme:/etc/caddy/acme # to save acme
-      - ./certs/ocsp:/etc/caddy/ocsp # to save certificates
+      - ./caddy-info:/data/caddy
```

Comment:
Let's add a comment on why do we need this line

Reply:
:heavy_check_mark: